core/templates/dev/head/exploration_editor/exploration_editor.html

```diff
@@ -179,7 +179,7 @@
     <li ng-if="isEditableOutsideTutorialMode()">
       <div dropdown class="btn-group" style="margin-right: 10px; margin-top: 8px;"
            title="<[getSaveButtonTooltip()]>">
-        <button id="tutorialSaveButton" class="btn btn-default oppia-save-draft-button protractor-test-save-changes" ng-class="{'btn-success': isExplorationSaveable()}" ng-click="saveChanges()" ng-disabled="!isExplorationSaveable()">
+        <button id="tutorialSaveButton" class="btn btn-default oppia-save-draft-button protractor-test-save-changes" style="padding-right: 130px;" ng-class="{'btn-success': isExplorationSaveable()}" ng-click="saveChanges()" ng-disabled="!isExplorationSaveable()">
```

Comment:
Yep.
I added because the number of changes were not visible because number was hidden behind the carat symbol.
So I added the style to the span of ``` publish changes ``` but then it went wrong since it made the number go far away from text.
But still II'l try to think of better solution 
Thanks!